### ISSUE TITLE
neovim support

### ISSUE THREAD

**Issue by nhooyr** [ludovicchabant/vim-gutentags#66]
Would be nice to have the status line update itself once the tag file generation is done. And it would not require the lock file.

**Comment by justinmk** [comment]
Showing a message when ctags is done would be very annoying--it rebuilds tags often. I think you'll find that you don't ever need to think about whether the tags have built--that's what makes the plugin nice.

**Comment by justinmk** [comment]
That being said, the lockfile _is_ annoying, especially given that it's not needed at all with neovim (nor Vim itself since ~7.4.1300). The lockfile shows up in `:Gstatus`, for one thing.

**Comment by nhooyr** [comment]
@justinmk not a message, I meant in the status line. There is already the function` gutentags#statusline()` but it doesn't update itself. You have to trigger a status line update by moving around or something.

**Comment by ludovicchabant** [comment]
Heya! So yes, I started looking into the job functions of the newest Vim version, and I'll definitely use it for Gutentags -- I just don't know when yet. But yeah, it will be good to get rid of the lock file!

I don't use Neovim personally so I won't do any testing or development specifically for it though, so that's that.
